packages/office-ui-fabric-react/src/components/Pivot/Pivot.styles.ts

```diff
@@ -17,19 +122,95 @@ export const getStyles = (props: IPivotStyleProps): IPivotStyles => {
         color: palette.themePrimary,
         whiteSpace: 'nowrap'
       },
+      rootIsLarge && [classNames.rootIsLarge],
+      rootIsTabs && [classNames.rootIsTabs],
       className
     ],
-
-    links: ['ms-Pivot-links', {}],
-
-    link: ['ms-Pivot-link', {}],
-
-    text: ['ms-Pivot-text', {}],
-
-    count: ['ms-Pivot-count', {}],
-
-    icon: ['ms-Pivot-icon', {}],
-
-    ellipsis: ['ms-Pivot-ellipsis', {}]
+    link: [
+      ...linkStyles(props, classNames),
+      rootIsTabs && {
+        selectors: {
+          '&:hover, &:focus': {
+            color: palette.black,
+            zIndex: 1
+          },
+          '&:active': {
+            color: palette.white,
+            backgroundColor: palette.themePrimary
+          }
+        }
+      }
+    ],
+    linkIsSelected: [
+      classNames.isSelected,
+      ...linkStyles(props, classNames),
+      {
+        fontWeight: FontWeights.semibold,
+        selectors: {
+          // TODO: not appearing in DOM / not working
+          ':before': {
+            boxSizing: 'border-box',
+            borderBottom: `2px solid ${palette.themePrimary}`
```

Comment:
I need to figure this out too. The underline isn't appearing but I haven't looked too deeply into it yet. If it's obvious to anybody let me know

Reply:
display: block? or does position absolute assume display block?

Reply:
Turns out it's because there was no content. Content needs to be ``` `""` ``` instead of ``` `` ```